optimade/server/main.py

```diff
@@ -70,17 +70,15 @@ def load_entries(endpoint_name: str, endpoint_collection: MongoCollection):
 app.add_exception_handler(NotImplementedError, exc_handlers.not_implemented_handler)
 app.add_exception_handler(Exception, exc_handlers.general_exception_handler)
 
+# Add various endpoints to unversioned URL
+for endpoint in (info, links, references, structures, landing, versions):
+    app.include_router(endpoint.router)
 
-# Add various endpoints to `/vMAJOR`
-app.include_router(info.router, prefix=BASE_URL_PREFIXES["major"])
-app.include_router(links.router, prefix=BASE_URL_PREFIXES["major"])
-app.include_router(references.router, prefix=BASE_URL_PREFIXES["major"])
-app.include_router(structures.router, prefix=BASE_URL_PREFIXES["major"])
 
-
-# Add the router for the landing page for all prefixes
-app.include_router(landing.router)
-app.include_router(landing.router, prefix=BASE_URL_PREFIXES["major"])
+def add_major_version_base_url(app: FastAPI):
+    """ Add mandatory vMajor endpoints, i.e. all except versions. """
```

Comment:
```suggestion
    """ Add mandatory vMajor endpoints, excluding 'versions'. """
```

Reply:
I wrote this myself first, but on re-reading it sounded like `versions` was a mandatory `vMajor` endpoint, which it isn't!